build_rules.gradle

```diff
@@ -263,9 +263,8 @@ ext.applyJavaNature = {
   apply plugin: "com.diffplug.gradle.spotless"
   spotless {
     java {
-      target rootProject.fileTree(rootProject.rootDir) {
-        include 'sdks/java/**/*.java'
-      }
+      target "src/**/*.java"
```

Comment:
It seems as though the default is already all java src based upon:
https://github.com/diffplug/spotless/tree/master/plugin-gradle#applying-to-java-source

Maybe we can just drop the entire java block.